interfaces/matlab/Utility/geterr.m

```diff
@@ -4,8 +4,8 @@
     checklib;
     try
         buflen = calllib(ct, 'ct_getCanteraError', 0, '');
-        aa = char([zeros(1, buflen, 'int8')]);
-        ptr = libpointer('cstring', aa);
+        aa = zeros(1, buflen+1, 'int8');
+        ptr = libpointer('voidPtr', aa);
```

Comment:
I think these two lines are unrelated to the rest of the commit, and part of our other efforts to debug the string passing?  not a big deal, but in general best to keep each commit to one "topic"

Reply:
Sorry I was supposed to stash this change but rather got included in the commit.